### PR TITLE
PMM-4658 use release repository for images builds

### PR DESCRIPTION
**Pull request by hors**
https://pmm.cd.percona.com/job/pmm-submodules/job/PR-448/

### CODE REVIEW

File: ansible/roles/pmm2-images/vars/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # !!! WARNING _experimental_ has to be replaced by _release_ after GA
-    pmm2_server_repo: laboratory
+    pmm2_server_repo: release
     pmm_client_repos: "original testing"
```
[the remark this answers is not in the document]
Comment (hors) [reply ansible/roles/pmm2-images/vars/main.yml]: It is the name of this repo: https://repo.percona.com/yum/testing/7/RPMS/x86_64/  

yes, we use it in https://github.com/Percona-Lab/percona-images/blob/6629511a76bf86adb6dcad2c01ae239b64e9f735/ansible/roles/pmm2-images/tasks/main.yml#L115

[the remark this answers is not in the document]
Comment (AlekSi) [reply ansible/roles/pmm2-images/vars/main.yml]: We should use `release`. We will push pmm2-client rc1 to `release`. @EvgeniyPatlan please confirm.

File: ansible/roles/pmm2-images/vars/main.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
     pmm_client_repo_name: "percona-testing-x86_64"
```
[the remark this answers is not in the document]
Comment (hors) [reply ansible/roles/pmm2-images/vars/main.yml]: it is the name of the following repo: https://repo.percona.com/yum/testing/7/RPMS/x86_64/

File: ansible/roles/pmm2-images/vars/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hors) [reply ansible/roles/pmm2-images/vars/main.yml]: @EvgeniyPatlan  maybe it should be experimental repo ?

[the remark this answers is not in the document]
Comment (EvgeniyPatlan) [reply ansible/roles/pmm2-images/vars/main.yml]: @AlekSi as I remember during our last call we agreed to use experimental for rc, am I right?

[the remark this answers is not in the document]
Comment (hors) [reply ansible/roles/pmm2-images/vars/main.yml]: @AlekSi please comment.

**Pull request merged**